shardingsphere-distsql-parser/shardingsphere-distsql-parser-engine/src/main/java/org/apache/shardingsphere/distsql/parser/core/DistSQLVisitor.java

```diff
@@ -372,7 +373,12 @@ public ASTNode visitShowResources(final ShowResourcesContext ctx) {
     public ASTNode visitShowRule(final ShowRuleContext ctx) {
         return new ShowRuleStatement(ctx.ruleType().getText(), null == ctx.schemaName() ? null : (SchemaSegment) visit(ctx.schemaName()));
```

Comment:
Could we put `String schemaName` in `ShowRuleStatement` instead of `SchemaSegment`?

Reply:
Yes, but there are too many classes associated with it, I will fix it in next pr.